docs/source/examples/function-calling.rst

```diff
@@ -64,16 +64,18 @@ The ``service.py`` file outlines the logic of the two required BentoML Services.
 
 1. Begin by specifying the LLM for the project. This example uses `Llama 3.1 70B Instruct AWQ in INT4 <https://huggingface.co/hugging-quants/Meta-Llama-3.1-70B-Instruct-AWQ-INT4>`_ and you may choose an alternative model as needed.
 
-   .. code-block:: bash
+   .. code-block:: python
+      :caption: `service.py`
 
-    	MODEL_ID = "hugging-quants/Meta-Llama-3.1-70B-Instruct-AWQ-INT4"
+      MODEL_ID = "hugging-quants/Meta-Llama-3.1-70B-Instruct-AWQ-INT4"
 
 2. Create a Python class (``Llama`` in the example) to initialize the model and tokenizer, and use the following decorators to add BentoML functionalities.
 
    - ``@bentoml.service``: Converts this class into a BentoML Service. You can optionally set :doc:`configurations </reference/bentoml/configurations>` like timeout and GPU resources to use on BentoCloud. We recommend you use an NVIDIA A100 GPU of 80 GB for optimal performance.
    - ``@bentoml.asgi_app``: Mounts an `existing ASGI application <https://github.com/bentoml/BentoFunctionCalling/blob/main/openai_endpoints.py>`_ defined in the ``openai_endpoints.py`` file to this class. It sets the base path to ``/v1``, making it accessible via HTTP requests. The mounted ASGI application provides OpenAI-compatible APIs and can be served side-by-side with the LLM Service. For more information, see :doc:`/build-with-bentoml/asgi`.
 
    .. code-block:: python
+      :caption: `service.py`
 
       import bentoml
       from openai_endpoints import openai_api_app
```

Comment:
Theres no openai_endpoints here anymore

Reply:
I can still see we are using the file here: https://github.com/bentoml/BentoFunctionCalling/blob/main/openai_endpoints.py

And it is used in `service.py` 

```python
from openai_endpoints import openai_api_app

...

@bentoml.mount_asgi_app(openai_api_app, path="/v1")
```

@larme Do we want to update the code here?

Reply:
oh I see this is using lmdeploy. Then we can keep it as it then

Reply:
No need to change anything then